src/app/public/modules/selection-box/fixtures/selection-box-fixtures.module.ts

```diff
@@ -45,7 +46,8 @@ import {
     SkyIconModule,
     SkyIdModule,
     SkyRadioModule,
-    SkySelectionBoxModule
+    SkySelectionBoxModule,
+    ReactiveFormsModule
```

Comment:
Imported modules should be alphabetically arranged.

Reply:
Fixed in last commit